tests/modules/mcuboot/direct_xip/src/main.c

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <ztest.h>
+#include <stdio.h>
+
+
+void test_direct_xip(void)
+{
+	zassert_true(true, "Needed to pass.");
```

Comment:
Please add here a comment, why this test looks like this and why you pass `true` to `zassert_true()` assert. I guess that in this tests you only check if this program is able to boot and run this test, but in my opinion it need to be explain more precisely.

Reply:
done

Reply:
Thanks :)

Reply:
thanks, this is fixed now